examples/catalog/test/custom_semantics_test.dart

```diff
@@ -18,7 +18,7 @@ void main() {
     await tester.pump();
 
     // Verify it correctly exposes its semantics.
-    // TODO(goderbauer): Use `SemanticsTester` after https://github.com/flutter/flutter/issues/12286.
+    // Change what?
```

Comment:
@goderbauer can you advise what change should be made here now that this issue is resolved?

Reply:
This should be using WidgetTesting.getSemantics [1].

The line below should say something like 

```dart
final SemanticsNode semantics = tester.getSemantics(find.byType(AdjustableDropdownListTile));
```

[1] https://master-api.flutter.dev/flutter/flutter_test/WidgetTester/getSemantics.html